include/glow/Graph/VerifierHelper.h

```diff
@@ -135,6 +135,41 @@ bool expectCompareTrue(
   return false;
 }
 
+/// Check whether $V_{0,n}{comp(\p a, \p b_i)}$ is true.
+/// If that check fails, \p msg is printed out using glow::report
+/// and \p parent (if not nullptr), \p a, and \p b are printed out
+/// using glow::reportContext.
+/// \returns \p comp(\p a, \p b_0) v ... v comp(\p a, \p b_i).
+template <typename InputTy>
+bool expectCompareTrue(
+    const char *msg, const InputTy &a, llvm::ArrayRef<InputTy> b,
+    const Node *parent,
+    const CompareWithName<InputTy> &comp = CompareOperatorEqual<InputTy>()) {
+  bool seed = false;
```

Comment:
can you find a better naming result or something?

Reply:
How about `result`?